src/Main.java

```diff
@@ -1,61 +1,144 @@
+        double a = 27.12;
+        long b = 987678965549L;
+        float c = 2.786f;
+        short d = 569;
+        short e = -159;
+        short n = 27897;
+        byte k = 67;
+        System.out.println();
+        System.out.println("Задание 3");
+        byte LyudmilaPavlovna = 23;
+        byte AnnaSergeevna = 27;
+        byte EkaterinaAndreevna = 30;
+        short totalSheetsOfPaper = 480;
+        short totalStudents = (short) (LyudmilaPavlovna + AnnaSergeevna + EkaterinaAndreevna);
+        byte sheetsForEachStudent = (byte) (totalSheetsOfPaper / totalStudents);
+        System.out.println("На каждого ученика рассчитано " + sheetsForEachStudent + " листов бумаги");
+
+        System.out.println();
+        System.out.println("Задание 4");
+        byte MachinePerformanceIn2Minutes = 16;
+        byte MachinePerformanceIn1Minutes = (byte) 16 / 2;
+        byte TwentyMinutes = 20;
+        short MachinePerformanceIn20Minutes = (short) (MachinePerformanceIn1Minutes * TwentyMinutes);
+        System.out.println("За " + TwentyMinutes + " машина произвела " + MachinePerformanceIn20Minutes + " штук");
+
+        byte hoursInADday = 24;
+        byte howManyMinutesInOneHour = 60;
+        short perDayMinutes = (short) (hoursInADday * howManyMinutesInOneHour);
+        short MachinePerformanceInPerDay = (short) (MachinePerformanceIn1Minutes * perDayMinutes);
+        System.out.println("В сутки" + " машина произвела " + MachinePerformanceInPerDay + " штук");
+
+        byte NumberOfDays = 3;
+        short inThreeDaysMinutes = (short) (NumberOfDays * perDayMinutes);
+        int MachinePerformanceinThreeDays = MachinePerformanceIn1Minutes * inThreeDaysMinutes;
+        System.out.println("За 3 дня" + " машина произвела " + MachinePerformanceinThreeDays + " штук");
+
+        byte totalDaysInOneMonth = 30;
+        int inOneMonthOnlyMinutes = totalDaysInOneMonth * perDayMinutes;
+        int inOneMonth = inOneMonthOnlyMinutes * MachinePerformanceIn1Minutes;
+        System.out.println("За 1 месяц" + " машина произвела " + inOneMonth + " штук");
+
+        System.out.println();
+        System.out.println("Задание 5");
+        byte totalColors = 120;
+        byte oneClassWhite = 2;
+        byte oneClassBrown = 4;
+        byte totalPaintsForOneClass = (byte) (oneClassBrown + oneClassWhite);
+        byte totalClass = (byte) (totalColors / totalPaintsForOneClass);
+        byte classWhite = (byte) (oneClassWhite * totalClass);
+        byte classBrown = (byte) (oneClassBrown * totalClass);
+        System.out.println("В школе, где " + totalClass + " классов, нужно " + classWhite + " банок белой краски и "
+                + classBrown + " банок коричневой краски");
+        System.out.println();
+        System.out.println("Задание 6");
+        byte bananas = 5;
+        byte weightOfOneBanana = 80;
+        short totalWeightBanana = (short) (bananas * weightOfOneBanana);
+
+        short TotalMilkInLiters = 200;
```

Comment:
```suggestion
        short totalMilkInLiters = 200;
```
названия переменных надо писать с маленькой буквы